app/process/watcher/message/watcher.go

```diff
@@ -39,21 +39,48 @@ type Watcher struct {
 	topicID          hedera.TopicID
 	statusRepository repository.Status
 	pollingInterval  time.Duration
-	startTimestamp   int64
 	logger           *log.Entry
 }
 
-func NewWatcher(client client.MirrorNode, topicID string, repository repository.Status, pollingInterval time.Duration, startTimestamp int64) *Watcher {
+func NewWatcher(
+	client client.MirrorNode,
+	topicID string,
+	repository repository.Status,
+	pollingInterval time.Duration,
+	startTimestamp int64) *Watcher {
 	id, err := hedera.TopicIDFromString(topicID)
 	if err != nil {
 		log.Fatalf("Could not start Consensus Topic Watcher for topic [%s] - Error: [%s]", topicID, err)
 	}
 
+	targetTimestamp := time.Now().UnixNano()
```

Comment:
I think that this property is not used anywhere

Reply:
`Create`